dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+https://github.com/cloudify-cosmo/cloudify-dsl-parser@master#egg=cloudify-dsl-parser==3.2
--e git+https://github.com/cloudify-cosmo/cloudify-rest-client@master#egg=cloudify-rest-client==3.2
--e git+https://github.com/cloudify-cosmo/cloudify-plugins-common@master#egg=cloudify-plugins-common==3.2
+-e git+https://github.com/cloudify-cosmo/cloudify-dsl-parser@3.2#egg=cloudify-dsl-parser==3.2
```

Comment:
On master branch should remain master (as opposed to build branches and tags where master is replaced with a specific version).